google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/spi/v2/HttpBigQueryRpc.java

```diff
@@ -276,13 +297,23 @@ public boolean deleteDataset(String projectId, String datasetId, Map<Option, ?>
   public Dataset patch(Dataset dataset, Map<Option, ?> options) {
     try {
       validateRPC();
+      Integer accessPolicyVersion = null;
```

Comment:
Ditto regarding moving declaration of accessPolicyVersion closer to usage.

Reply:
Fixed.